src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -302,6 +309,17 @@ public FlutterLogTree(@NotNull FlutterApp app, @NotNull Disposable parent) {
     this.model = model;
   }
 
+  public void addListener(@NotNull EventCountListener listener, @Nullable Disposable parent) {
```

Comment:
Do we repeat `com.intellij.util.EventDispatcher` here?
It seems quite popular in IntelliJ code.

Reply:
Oh!  Handy.  Thanks!